client/verta/verta/_registry/modelversion.py

```diff
@@ -313,6 +345,24 @@ def _create_artifact_msg(self, key, artifact_stream, artifact_type, extension=No
 
         return artifact_msg
 
+    def _get_artifact(self, key, artifact_type):
+        # check to see if key exists
+        self._refresh_cache()
+        if key == "model":
+            # get model artifact
+            if not self.has_model:
+                raise KeyError("no model associated with this version")
+        elif len(filter(lambda artifact: artifact.key == key, self._msg.artifacts)) == 0:
```

Comment:
`filter` isn't `len`able in Python 3.
I now realize the specific gotchas that we're going to have moving forward.
```suggestion
        elif len(list(filter(lambda artifact: artifact.key == key, self._msg.artifacts))) == 0:
```

Reply:
Updated.